tests/meltano/core/test_environment_variables.py

```diff
@@ -166,72 +166,54 @@ def _meltanofile_update_dict(
     ),
     # Original xfail'ing tests, as per comment above
```

Comment:
@cjohnhanson should this comment and the one above be removed?

Reply:
Yep, along with one other comment. I also removed the `xfail` attribute from these test cases since it's no longer  used.